clients/fides-js/src/components/ConsentBanner.tsx

```diff
@@ -94,10 +95,7 @@ const ConsentBanner: FunctionComponent<BannerProps> = ({
                   {bannerTitle}
                 </div>
                 {showGpcBadge && (
-                  <GpcBadge
-                    label="Global Privacy Control Signal"
-                    status="detected"
-                  />
+                  <GpcBadge i18n={i18n} status={GpcStatus.APPLIED} />
```

Comment:
Modified this _slightly_ to not invent a new "Detected" status type and reuse the same translation.